test/end-to-end/core.sh

```diff
@@ -480,6 +508,42 @@ os::cmd::expect_success "oc exec -p ${registry_pod} du /registry > '${LOG_DIR}/p
 
 # make sure there were changes to the registry's storage
 os::cmd::expect_code "diff ${LOG_DIR}/prune-images.before.txt ${LOG_DIR}/prune-images.after.txt" 1
+echo "[INFO] Validated image pruning"
+
+echo "[INFO] Configure registry to accept manifest V2 schema 2"
+os::cmd::expect_success "oc project ${CLUSTER_ADMIN_CONTEXT}"
+os::cmd::expect_success 'oc env -n default dc/docker-registry REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ACCEPTSCHEMA2=true'
+wait_for_registry
+echo "[INFO] Registry configured to accept manifest V2 schema 2"
+
+echo "[INFO] Accept manifest V2 schema 2"
+os::cmd::expect_success "oc login -u schema2-user -p pass"
+os::cmd::expect_success "oc project schema2"
+# tagging remote docker.io/busybox image
+os::cmd::expect_success "oc tag --source=docker busybox '${DOCKER_REGISTRY}/schema2/busybox'"
```

Comment:
actually, I was wrong... I think this was supposed to be `docker tag`, but without the `--source`